core/ui/EditTemplate/fields.tid

```diff
@@ -76,57 +68,46 @@ $value={{{ [<newFieldValueTiddler>get[text]] }}}/>
 
 <$fieldmangler>
 <div class="tc-edit-field-add">
-<em class="tc-edit">
-<<lingo Fields/Add/Prompt>>&nbsp;&nbsp;
+<em class="tc-edit tc-big-gap-right">
```

Comment:
This should have been a `tc-small-gap-right` see: default edit-template has a visual problem #5424

@Jermolene .. The types input also got a "big-gap", which doesn't create a problem there. Should we also change it to "small"